sdk/core/Azure.Core.TestFramework/src/PlaybackTransport.cs

```diff
@@ -66,6 +66,9 @@ public override async ValueTask ProcessAsync(HttpMessage message)
                 message.Request.Content.TryComputeLength(out long length) &&
                 length > 0)
             {
+                // Set a Content-Length header if one doesn't exist
```

Comment:
This doesn't do exactly what the comment says/original logic did. The content-lenght header value is overridden unconditionally.

Reply:
Good catch, thanks